.gitignore

```diff
@@ -20,6 +20,8 @@ _build
 *.annot
 *.mj
 *.bak
+*.swp
+*.swo
```

Comment:
Oh, those are VIM swap files aren't they?  They should probably be in your global gitignore definition, not per-project.  I'd vote on removing these as they are IDE dependent and not part of the build system.